model/note/image.go

```diff
@@ -178,6 +178,37 @@ func (u *ImageUpload) Close() error {
 	return nil
 }
 
+// CopyImageToAnotherNote makes a copy of an image from one note to be used in
+// another note.
+func CopyImageToAnotherNote(inst *instance.Instance, srcDoc *vfs.FileDoc, imageID string, dstDoc *vfs.FileDoc) (*Image, error) {
```

Comment:
What's the `srcDoc` argument used for here?

Reply:
Good question, and the response is nothing. I can remove it.